Sources/PackageModel/Product.swift

```diff
@@ -241,26 +246,3 @@ extension ProductType: Codable {
         }
     }
 }
-
-extension ProductFilter: Codable {
```

Comment:
The old `Codable` implementation here could not be added to in a backwards compatible way. I just removed it to let compiler synthesize it.

I do not know where or if this is actually used. We stopped recording filters in the pins file a while ago. Changing it did not flag on any tests. Do you think there is anywhere we need to bump a format version?

Reply:
I don't think this is stored anyway, though @neonichu or @tomerd would know better.